cherab/solps/formats/balance.py

```diff
@@ -0,0 +1,165 @@
+        if fhandle.variables['species'].data[i,1] == b'N' and fhandle.variables['species'].data[i,2] == b'e':
+            zn = 10
+        if fhandle.variables['species'].data[i,1] == b'A' and fhandle.variables['species'].data[i,2] == b'r':
+            zn = 18
+	
+        am = float(fhandle.variables['am'].data[i])  # Atomic mass
+        charge = int(fhandle.variables['za'].data[i])  # Ionisation/charge
+        species = _popular_species[(zn, am)]
+		
+        # If we only need to populate species_list, there is probably a faster way to do this...		
+        sim.species_list.append(species.symbol + str(charge))
+
+    tmp = fhandle.variables['na'].data.copy()
+    tmp = np.moveaxis(tmp, 0, -1)
+    sim._species_density = tmp
+
+    # Make Mesh Interpolator function for inside/outside mesh test.
+    inside_outside_data = np.ones(mesh.num_tris)
+    inside_outside = AxisymmetricMapper(Discrete2DMesh(mesh.vertex_coords, mesh.triangles, inside_outside_data, limit=False))
+    sim._inside_mesh = inside_outside
+
+    
+    # Load the neutrals data
+    D0_indx = sim.species_list.index('D0')
+
+    # Replace the deuterium neutrals density (from the fluid neutrals model by default) with
+    # the values calculated by EIRENE - do the same for other neutrals?
+    if 'dab2' in fhandle.variables.keys():
+        if D0_indx is not None:
+            b2_len = np.shape(sim.species_density[:,:,D0_indx])[-1]
+            eirene_len = np.shape(fhandle.variables['dab2'].data)[-1]
+            sim.species_density[:,:,D0_indx] = fhandle.variables['dab2'].data[0,:,0:b2_len-eirene_len]
+	
+        eirene_run = True
+    else:
+        eirene_run = False
+    
+    # Calculate the total radiated power
+    
+    if eirene_run:
+        # Total radiated power from B2, not including neutrals
+        b2_ploss = fhandle.variables['b2stel_she_bal'].data.copy()/vol
+        
+        # Electron energy loss due to interactions with neutrals
+        if 'eirene_mc_eael_she_bal' in fhandle.variables.keys():
+            eirene_ecoolrate = np.sum(fhandle.variables['eirene_mc_eael_she_bal'].data,axis=0)/vol
+            
+        # Ionisation rate from EIRENE, needed to calculate the energy loss to overcome the ionisation potential of atoms
+        if 'eirene_mc_papl_sna_bal' in fhandle.variables.keys():
+            eirene_potential_loss = 13.6*np.sum(fhandle.variables['eirene_mc_papl_sna_bal'].data,axis=(0))[1,:,:]*Q/vol
+        
+        # This will be negative (energy sink); multiply by -1
+        sim._total_rad = -1.0*(b2_ploss+(eirene_ecoolrate-eirene_potential_loss))
+        
+    else:
+ 
+        # Total radiated power from B2, not including neutrals
+        b2_ploss = np.sum(fhandle.variables['b2stel_she_bal'].data,axis=0)/vol
+        
+        potential_loss = np.sum(fhandle.variables['b2stel_sna_ion_bal'].data,axis=0)/vol
+        
+        # Save total radiated power to the simulation object
+        sim._total_rad = 13.6*Q*potential_loss-b2_ploss
```

Comment:
Please add spaces around the multiplication and subtraction operators here, as well as the division operators in the lines above. Pylint will tell you if you've missed any.